src/jmcomic/jm_plugin.py

```
@@ -799,6 +799,90 @@
+
+        # 长图路径
+        long_img_path = os.path.join(img_dir, f'{filename}.png')
+
+        # 调用 PIL 把 photo_dir 下的所有图片合并为长图
+        img_path_ls = self.write_img_2_long_img(long_img_path, album, photo)
+        self.log(f'Convert Successfully: JM{album or photo} → {long_img_path}')
+
+        # 执行删除
+        self.execute_deletion(img_path_ls)
+
+    def write_img_2_long_img(self, long_img_path, album: JmAlbumDetail, photo: JmPhotoDetail) -> List[str]:
```

Ruff

`JmAlbumDetail` may be undefined, or defined from star imports

(F405)